app/models/user.rb

```diff
@@ -9,6 +9,14 @@ def validate record
 
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
```

Comment:
tương tự ở trên, check 1 lượt nhé em 
```suggestion
  has_many :active_relationships, class_name: Relationship.name,
```